pkg/shed/schema.go

```diff
@@ -147,17 +147,22 @@ func (db *DB) schemaIndexPrefix(name string) (id byte, err error) {
 // getSchema retrieves the complete schema from
 // the database.
 func (db *DB) getSchema() (s schema, err error) {
+	if db.sch != nil {
+		return *db.sch, nil
+	}
 	b, err := db.Get(keySchema)
 	if err != nil {
 		return s, err
 	}
 	err = json.Unmarshal(b, &s)
+	db.sch = &s
```

Comment:
same here

Reply:
Again, since we're just remembering the schema, I don't see any chance for a panic here.